scripts/always.sh

```diff
@@ -3,6 +3,9 @@
 echo "starting httpd"
 systemctl start httpd.service
 
+echo "turning off firewalld"
+systemctl stop firewalld
```

Comment:
I agree that would be more thorough and it's what we used to do with the centos 6 boxes.

But, it's a lot of extra unnecessary work for the provisioners on a development box - our docs are quite explicit that the provisioners are not intended for production.

This line is ultimately only needed because this box comes with the firewall on whereas all the other 7.x boxes didn't.